package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "babel-cli": "^6.7.5",
     "babel-eslint": "^7.1.1",
+    "babel-plugin-transform-object-rest-spread": "6.22.0",
```

Comment:
I get this error without the dependency:

Error: Cannot find module 'babel-plugin-transform-object-rest-spread' (While processing preset: "[path...]/emojione-picker/node_modules/babel-preset-stage-0/lib/index.js")

Am I doing something else wrong?